t/Currency.t

```diff
@@ -455,58 +401,29 @@ ddg_spice_test(
         is_cached => 0
     ),
     # check if the currency is detected from the location
-    # when you don't specify the target currency
-    DDG::Request->new(
-        query_raw => "300 bgn",
-        location => test_location("de")
-    ) => test_spice(
-        '/js/spice/currency/300/bgn/eur',
-        call_type => 'include',
-        caller => 'DDG::Spice::Currency',
-        is_cached => 0
-    ),
-    DDG::Request->new(
-        query_raw => "6008 usd",
-        location => test_location("my")
-    ) => test_spice(
-        '/js/spice/currency/6008/usd/myr',
-        call_type => 'include',
-        caller => 'DDG::Spice::Currency',
-        is_cached => 0
-    ),
+    # when you don't specify the target Currency
     # check for placing currency type before amount
     'CHF 2.95 in eur' => test_spice(
         '/js/spice/currency/2.95/chf/eur',
         call_type => 'include',
         caller => 'DDG::Spice::Currency',
         is_cached => 0
     ),
-    'euro 400' => test_spice(
-        '/js/spice/currency/400/eur/usd',
-        call_type => 'include',
-        caller => 'DDG::Spice::Currency',
-        is_cached => 0
-    ),
     'what is rub 400 in us dollars?' => test_spice(
         '/js/spice/currency/400/rub/usd',
         call_type => 'include',
         caller => 'DDG::Spice::Currency',
         is_cached => 0
     ),
-
-    # language based queries
-    'currency converter' => test_spice(
-        '/js/spice/currency/100/usd/eur',
-        call_type => 'include',
-        caller => 'DDG::Spice::Currency',
-        is_cached => 0
-    ),
-    'convert currencies' => test_spice(
-        '/js/spice/currency/100/usd/eur',
-        call_type => 'include',
-        caller => 'DDG::Spice::Currency',
-        is_cached => 0
-    ),
+    
+    # should no longer work since deprecating the tile view
+    'canada dollar' => undef,
+    '400 euro' => undef,
+    '4k euro' => undef,
+    'bitcoin value' => undef,
+    'bitcoin price' => undef,
+    '100cad' => undef,
+    '100,000cad' => undef,
```

Comment:
I think these queries should continue to work, defaulting to a comparison with the user's currency:

- https://encrypted.google.com/search?hl=en&q=400%20euro
- https://encrypted.google.com/search?hl=en&q=bitcoin%20price
- https://encrypted.google.com/search?hl=en&q=canada%20dollar